packages/wallet/ui/src/util/tests/SuggestChain.test.js

```diff
@@ -0,0 +1,186 @@
+// @ts-check
+import { makeChainInfo } from '../SuggestChain.js';
+
+/**
+ * fetched 2022-10-27 from https://main.agoric.net/network-config
+ *
+ * @type {import('@agoric/internal').NetworkConfig}
+ */
+const mainnetExample = {
+  chainName: 'agoric-3',
+  gci: 'edef208d407403e6f478bc33dcf7bb294febc5bc6a423c720f0c9c50da5176d5',
+  peers: [
+    'a26158a5cbb1df581dd6843ac427191af76d6d5d@104.154.240.50:26656',
+    '6e26a1b4afa6889f841d7957e8c2b5d50d32d485@95.216.53.26:26656',
+  ],
+  rpcAddrs: ['https://agoric-rpc.polkachu.com:443'],
+  apiAddrs: ['https://agoric-api.polkachu.com:443'],
+  // @ts-expect-error invalid
+  oldRpcAddrs: ['https://main.rpc.agoric.net:443'],
+  seeds: [],
+};
+
+/**
+ * fetch 2022-10-27 from https://ollinet.agoric.net/network-config
+ *
+ * @type {import('@agoric/internal').NetworkConfig}
+ */
+const ollinetExample = {
+  chainName: 'agoricollinet-44',
+  gci: 'https://ollinet.rpc.agoric.net:443/genesis',
+  peers: ['fb86a0993c694c981a28fa1ebd1fd692f345348b@34.67.167.8:26656'],
+  rpcAddrs: ['https://ollinet.rpc.agoric.net:443'],
+  seeds: ['0f04c4610b7511a64b8644944b907416db568590@34.172.146.145:26656'],
+};
+
+const io = { random: Math.random };
```

Comment:
We have only 1 entry in our `rpcAddrs` array, so it doesn't matter yet, but this should be a mock random number generator with predictable outputs for testing.